packages/streamline/src/components/cards/card-carousel/card-carousel.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import { usePress } from '../../../hooks/use-press.hook';
+import { Box } from '../../../primitives/box/box';
+import { Card } from '../../../primitives/card/card';
+import { Icon } from '../../../primitives/icon/icon';
+import { Text } from '../../../primitives/text/text';
+import ButtonIcon from '../../buttons/button-icon/button-icon';
+import Spinner from '../../spinner/spinner';
+import Tag from '../../tag/tag';
+import { CardCarouselSkeleton } from './card-carousel-skeleton';
+import { CardCarouselProps } from './card-carousel.types';
+
+export const CardCarousel = (props: CardCarouselProps) => {
+  const {
+    size = 'LG',
+    iconName = 'Area',
```

Comment:
We do not want to have a default icon. It can be optional i guess.

Reply:
Changed props of CardCarousel. The iconName props is not required anymore.